pr-check.sh

```diff
@@ -43,18 +71,32 @@ function remove_venv_venv {
 }
 
 function create_conda_venv {
-    conda create -n $1 --yes python=3.6
+    conda create -n $1 --yes python=3.8
 }
 
 function activate_conda_venv {
     source activate $1
     # Pip may update setuptools while installing BrainIAK requirements and
     # break the Conda cached package, which breaks subsequent runs.
     conda install --yes -f setuptools
+
+    if [[ "$is_della" == true ]]; then
+        # On della, we need to set LD_LIBRARY_PATH to the conda environment libs explicitly.
+        # This is because when importing tensorflow the system libstd++ is picked
+        # up instead of the conda one. This causes subsequent GLIB version errors
+        # when trying to load compiled modules
+        export LD_LIBRARY_PATH="$CONDA_PREFIX/lib:$LD_LIBRARY_PATH"
+    fi
+
 }
 
 function deactivate_conda_venv {
-    source deactivate
+
+    if [[ "$is_della" == true ]]; then
+        conda deactivate
+    else
+        source deactivate
```

Comment:
I don't think that this is della specific, it is generally encouraged to use conda activate and conda deactivate